client/extensions/woocommerce/app/store-stats/store-stats-referrer-title/style.scss

```diff
@@ -0,0 +1,37 @@
+.store-stats-referrer-title.card  {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+
+	.store-stats-referrer-title__stat {
+		margin-bottom: 12px;
+	}
+	
+	.store-stats-referrer-title__label {
+		color: $gray-dark;
+		display: block;
+		font-size: 11px;
+		letter-spacing: 0.1em;
+		text-transform: uppercase;
+	}
+	
+	.store-stats-referrer-title__value {
+		color: $blue-wordpress;
+		display: block;
+		font-size: 16px;
```

Comment:
Might need
```
word-wrap: break-word;
```
To prevent this situation

<img width="246" alt="screen shot 2018-05-16 at 4 33 54 pm" src="https://user-images.githubusercontent.com/1922453/40096882-27e573ba-5927-11e8-9328-d7245e791760.png">